environment.d.ts

```diff
@@ -0,0 +1,16 @@
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      GITHUB_TOKEN: string
+      ALGOLIA_ADMIN_KEY: string
+      ETHERSCAN_API_KEY: string
+      GATSBY_ALGOLIA_APP_ID: string
+      GATSBY_ALGOLIA_SEARCH_KEY: string
+      GATSBY_GITHUB_TOKEN_READ_ONLY: string
+      GATSBY_FUNCTIONS_PATH: string
+      CROWDIN_API_KEY: string
+    }
+  }
+}
+
+export {}
```

Comment:
Thanks for doing that, good idea.

We already have a `src/global.d.ts`. Would you mind in putting this in there?